ml4gw/waveforms/phenom_p.py

```diff
@@ -0,0 +1,731 @@
+from typing import Dict, Tuple
+
+import torch
+from torchtyping import TensorType
+
+from ..constants import MPC_SEC, MTSUN_SI, PI
+from .phenom_d import IMRPhenomD
+
+
+class IMRPhenomPv2(IMRPhenomD):
+    def __init__(self):
+        super().__init__()
+
+    def forward(
+        self,
+        fs: TensorType,
+        m1: TensorType,
+        m2: TensorType,
+        s1x: TensorType,
+        s1y: TensorType,
+        s1z: TensorType,
+        s2x: TensorType,
+        s2y: TensorType,
+        s2z: TensorType,
+        dist_mpc: TensorType,
+        tc: TensorType,
+        phiRef: TensorType,
+        incl: TensorType,
+        f_ref: float,
+    ):
+        """
+        m1 must be larger than m2.
```

Comment:
Let's add a good docstring here

Reply:
I think it would be good to update the docstrings of all the other waveforms as well, by also mentioning the units of the parameters. I did face some issues when running phenomD, not realizing the distance is in Mpc. 

Maybe I could update them all at once in a new PR?

Reply:
Sure. Let's do it after.